spec/javascript/packages/document-capture/components/document-capture-warning-spec.jsx

```diff
@@ -51,6 +51,7 @@ describe('DocumentCaptureWarning', () => {
     isFailedDocType,
     isFailedResult,
     isFailedSelfieLivenessOrQuality = false,
+    isFailedSelfieFacematch = false,
```

Comment:
Non-blocking nit: I think changing this to `isFailedSelfieFaceMatch` would be a capital idea.

Reply:
Ah, that is the only place in the codebase with `Facematch` capitalization. [Fixed](https://github.com/18F/identity-idp/pull/10042/commits/7208fb469e9b8c177ac1f793d47001f8910aae9a)!